src/editor/Components/HierarchyWindow.tsx

```diff
@@ -0,0 +1,28 @@
+import { RefObject } from 'react';
+import * as THREE from 'three';
+
+interface HierarchyWindowProps {
+  objects: { id: string; name: string; ref: RefObject<THREE.Mesh> | null }[];
```

Comment:
to jest zly typ. do `HierachyWindow` podajemy arraya `JSX.Element`. `objects` z parenta bedzie trzeba zmienic